wdl/PBCCSOnlySingleFlowcell.wdl

```diff
@@ -1,24 +1,38 @@
 version 1.0
 
+##########################################################################################
+## A workflow that performs CCS correction on PacBio HiFi reads from a single flow cell.
+## The workflow shards the subreads into clusters and performs CCS in parallel on each cluster.
+## Ultimately, all the corrected reads (and uncorrected) are gathered into a single BAM.
+## Various metrics are produced along the way.
+##########################################################################################
+
 import "tasks/PBUtils.wdl" as PB
 import "tasks/Utils.wdl" as Utils
-import "tasks/AlignReads.wdl" as AR
 import "tasks/Finalize.wdl" as FF
 
 workflow PBCCSOnlySingleFlowcell {
     input {
-        String gcs_input_dir
+        String raw_reads_gcs_bucket
```

Comment:
Hmm... this is a reasonable change, but makes me wonder about some side effects that may make quite a bit of work for ourselves.  Will we need to go through the longread-project-analysis repo and change all of our input JSONs to match?

Reply:
That's a good point. I'll take care of that.